common/lib/util/utils.js

```diff
@@ -3,6 +3,8 @@ var Utils = (function() {
 
 	function Utils() {}
 
+	Utils.modes = [ 'PRESENCE', 'PUBLISH', 'SUBSCRIBE', 'PRESENCE_SUBSCRIBE' ];
```

Comment:
Could this be documented, have a more qualified name or even be located somewhere else but utils? It seems to relate to `TB2d` in the features spec, but that took me a while to work out.